rest/oidc_api_test.go

```diff
@@ -2287,6 +2287,7 @@ func TestEventuallyReachableOIDCClient(t *testing.T) {
 			request := createOIDCRequest(t, sessionEndpoint, token)
 			response, err := http.DefaultClient.Do(request)
 			require.NoError(t, err, "Error sending request with bearer token")
+			defer func() { assert.NoError(t, response.Body.Close()) }()
```

Comment:
Doesn't each response body need closing? The defer will only get one of the 3 here